frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -1037,6 +1052,16 @@ export const ClusterServiceVersionDetails: React.FC<ClusterServiceVersionDetails
                   csv={props.obj}
                 />
               )}
+              {(deprecatedPackage.deprecation ||
+                deprecatedChannel.deprecation ||
+                deprecatedVersion.deprecation) && (
+                <DeprecatedOperatorWarningAlert
```

Comment:
nit:
```js
  <DeprecatedOperatorWarningAlert
    deprecatedPackage
    deprecatedChannel
    deprecatedVersion
    dismissible
  />
```

Reply:
> nit:
> 
> ```js
>   <DeprecatedOperatorWarningAlert
>     deprecatedPackage
>     deprecatedChannel
>     deprecatedVersion
>     dismissible
>   />
> ```

@jhadvig Actually, I believe this wouldn't compile since deprecatedPackage, deprecatedChannel, and deprecatedVersion are objects. The following approach works, but I prefer the former because it appears clearer to me :)
```
               <DeprecatedOperatorWarningAlert
                  {...{deprecatedPackage,
                  deprecatedChannel,
                  deprecatedVersion,
                  dismissible: true
                  }}
                />
```